app/screens/global_threads/threads_list/thread/thread_footer/thread_footer.tsx

```diff
@@ -0,0 +1,116 @@
+
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+
+import FormattedText from '@components/formatted_text';
+import UserAvatarsStack from '@components/user_avatars_stack';
+import {changeOpacity, makeStyleSheetFromTheme} from '@utils/theme';
+
+import type ThreadModel from '@typings/database/models/servers/thread';
+import type UserModel from '@typings/database/models/servers/user';
+
+export type Props = {
+    author?: UserModel;
+    currentUserId: string;
+    participants: UserModel[];
+    teammateNameDisplay: string;
+    testID: string;
+    thread: ThreadModel;
+    theme: Theme;
+};
+
+const getStyleSheet = makeStyleSheetFromTheme((theme: Theme) => {
+    return {
+        container: {
+            flexDirection: 'row',
+            alignItems: 'center',
+            minHeight: 40,
+        },
+        avatarsContainer: {
+            marginRight: 12,
+            paddingVertical: 8,
+        },
+        replyIconContainer: {
+            top: -1,
+            marginRight: 5,
+        },
+        replies: {
+            alignSelf: 'center',
+            color: changeOpacity(theme.centerChannelColor, 0.64),
+            fontSize: 12,
+            fontWeight: '600',
+            marginRight: 12,
+        },
+        unreadReplies: {
+            alignSelf: 'center',
+            color: theme.sidebarTextActiveBorder,
+            fontSize: 12,
+            fontWeight: '600',
+            marginRight: 12,
+        },
+    };
+});
+
+const ThreadFooter = ({author, currentUserId, participants, teammateNameDisplay, testID, theme, thread}: Props) => {
+    const style = getStyleSheet(theme);
+
+    let repliesComponent;
+    if (thread.unreadReplies) {
+        repliesComponent = (
+            <FormattedText
+                id='threads.newReplies'
+                defaultMessage='{count} new {count, plural, one {reply} other {replies}}'
+                style={style.unreadReplies}
+                testID={`${testID}.unread_replies`}
+                values={{
+                    count: thread.unreadReplies,
+                }}
+            />
+        );
+    } else if (thread.replyCount) {
+        repliesComponent = (
+            <FormattedText
+                id='threads.replies'
+                defaultMessage='{count} {count, plural, one {reply} other {replies}}'
+                style={style.replies}
+                testID={`${testID}.reply_count`}
+                values={{
+                    count: thread.replyCount,
+                }}
+            />
+        );
+    }
+
+    // threadstarter should be the first one in the avatars list
+    const participantsList = useMemo(() => {
+        if (author && participants?.length) {
+            const filteredParticipantsList = participants.filter((participant) => participant.id !== author.id).reverse();
+            filteredParticipantsList.unshift(author);
+            return filteredParticipantsList;
+        }
+        return [];
```

Comment:
~~Probably does not happen often, but if different renders of useMemo both return the empty list, this would be returning different empty lists. To avoid that, you can define this empty list outside of the component, and return the reference to that empty list.~~

I see that we won't be passing to any child component the list if it is empty, so it can be left like this.